common/djangoapps/util/organizations_helpers.py

```diff
@@ -103,3 +103,10 @@ def organizations_enabled():
     Returns boolean indication if organizations app is enabled on not.
     """
     return settings.FEATURES.get('ORGANIZATIONS_APP', False)
+
+
+def course_organization_linking_enabled():
```

Comment:
Instead of writing this new method modify `organizations_enabled` to add new waffle switch in its condition

Reply:
done